packages/config/src/FromConfigAdapter/FromConfigAdapter.ts

```diff
@@ -129,7 +133,7 @@ export default class FromConfigAdapter extends BaseAdapter {
   getFeatures(region: INoAssemblyRegion) {
     const { refName, start, end } = region
 
-    return ObservableCreate<Feature>(async observer => {
+    return ObservableCreate<SimpleFeature>(async observer => {
```

Comment:
Any reason why not just return with the Feature base class e.g. ObservableCreate<Feature>